php/public/index.php

```diff
@@ -12,6 +12,8 @@
 use Slim\Factory\AppFactory;
 use Slim\Views\Twig;
 use Slim\Views\TwigMiddleware;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\RequestInterface as Request;
```

Comment:
not completely sure if `ServerRequestInterface` would not be the better one and/or only for authentication handling... Would need to be tested.

Reply:
psalm added the solution